lib/code_corps/github/github.ex

```diff
@@ -1,6 +1,5 @@
 defmodule CodeCorps.GitHub do
```

Comment:
This file is split out a bit. Parts have been moved to `github/api.ex` and parts to `github/headers.ex`. Both these files probably need some form or testing, api more so.